environment.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+
+function checkEnvironmentForConfig(config:Object) : Object {
+  let mentionBotEnvConfig;
+
+  try {
+    mentionBotEnvConfig = JSON.parse(process.env.MENTION_BOT_CONFIG);
+  } catch(e) {
+    mentionBotEnvConfig = {};
+  }
+
+  return Object.keys(config).reduce((previousValue, key) => {
+    let defaultConfigValue = config[key];
+
+    let environmentVariable = mentionBotEnvConfig[key];
+    let configElement = {};
+    configElement[key] = environmentVariable === undefined ? defaultConfigValue
```

Comment:
An easier way to solve this is to use the spread operator to do that logic for you:

```js
{
  ...defaultConfig,
  ...customConfig
}
```

the `...` will override all the default values when a custom one exists